Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Python 3.8
+FROM python:3.9
+
+EXPOSE 8501
+
+# SHELL ["/bin/bash", "--login", "-c"]
+
+LABEL version="0.1.0"
```

Comment:
I think it's good to include a version number, however, GatorMiner hasn't had a release yet and I don't know what would be the correct number here. We should talk about it more before putting this here.

Reply:
I'm fine with having `0.1.0` as the first version for the docker image. I'm under the impression that @BennyWestsyde was using the datetime as a tag for the docker version in the CD. Probably a better idea to stick with the version number.